tests/test_enterprise/api/test_views.py

```diff
@@ -2385,6 +2385,52 @@ def test_post_licensed_course_enrollments_license_revoke(
         assert enterprise_course_enrollment.saved_for_later == is_revoked
         assert licensed_course_enrollment.is_revoked == is_revoked
 
+    @ddt.data(
+        # API should return enterprise customer details
+        {'is_staff': False, 'is_linked_to_enterprise': True, 'has_access': True},
+        # API should not return enterprise customer details
+        {'is_staff': False, 'is_linked_to_enterprise': False, 'has_access': False},
+        # for staff, API should return enterprise customer details irrespective of linked status
+        {'is_staff': True, 'is_linked_to_enterprise': True, 'has_access': True},
+        # for staff, API should return enterprise customer details irrespective of linked status
+        {'is_staff': True, 'is_linked_to_enterprise': False, 'has_access': True},
+    )
+    @ddt.unpack
+    def test_enterprise_customer_filter(self, is_staff, is_linked_to_enterprise, has_access):
```

Comment:
I think these tests would make sense in [test_filters.py](https://github.com/edx/edx-enterprise/blob/f872e39e18046e1de395fb1c29f1b7d38f0016c8/tests/test_enterprise/api/test_filters.py), where we have existing tests for filters (e.g., `UserFilterBackend`).